cachi2/core/package_managers/general.py

```diff
@@ -198,3 +201,14 @@ def extract_git_info(vcs_url: str) -> dict[str, Any]:
         "namespace": namespace,
         "repo": repo,
     }
+
+
+def get_repo_name_from_origin_remote(package_dir: RootedPath) -> str:
+    """Extract repositoy name from the git URL in the package directory."""
```

Comment:
Nitpick: Do you think it's worth stating in the docstring that this function can raise a `UnsupportedFeature` exception? (alternatively, we could treat the exception within the function's scope an return `Optional[str]`)

Reply:
Also s/repositoy/repository/

Reply:
I dropped that commit altogether; it would just require too much refactoring. Sorry for the noise.